angular-spa-ui/js/search-app/tags-module/tags-api-ctrl.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+module.exports =  function apiTagCtrl($scope, resultTagPageSrv) {
+
+    $scope.tags = {};
+    $scope.isEmpty = 'empty';
+    $scope.form = {
+        name: ""
+    };
+
+    // Visibility (show/hide) list tags
+    function isEmpty(tagsList) {
```

Comment:
Function that names from "is" can only return a boolean value. And it could not to be used as setter for anything.

function isEmpty(tagsList) {
return (tagsList.length == 0) ? true : false;
}